<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJemaat</class>
 <widget class="QWidget" name="FormJemaat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Jemaat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>&lt;h2&gt;Form Data Jemaat&lt;/h2&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupInput">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutInput">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelId">
            <property name="text">
             <string>ID Jemaat (kosong = auto)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="EditId"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelNama">
            <property name="text">
             <string>Nama Jemaat</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="EditNama"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelJK">
            <property name="text">
             <string>Jenis Kelamin</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="ComboJK">
            <item>
             <property name="text">
              <string>L</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>P</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelAlamat">
            <property name="text">
             <string>Alamat</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="EditAlamat"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelTempat">
            <property name="text">
             <string>Tempat Lahir</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="EditTempatLahir"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelTgl">
            <property name="text">
             <string>Tanggal Lahir</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="DateTglLahir">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelHp">
            <property name="text">
             <string>No HP</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="EditNoHP"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelStatusKeluarga">
            <property name="text">
             <string>Status Dalam Keluarga</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="EditStatusKeluarga"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelWilayah">
            <property name="text">
             <string>Wilayah</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="EditWilayah"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridButtons">
          <item row="0" column="0">
           <widget class="QPushButton" name="BtnSimpan">
            <property name="text">
             <string>SIMPAN</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="BtnUbah">
            <property name="text">
             <string>UBAH</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="BtnHapus">
            <property name="text">
             <string>HAPUS</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="BtnBersih">
            <property name="text">
             <string>BERSIH</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacerInput">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupTabel">
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutData">
        <item>
         <layout class="QHBoxLayout" name="layoutCariCetak">
          <item>
           <widget class="QLabel" name="labelCari">
            <property name="text">
             <string>Cari:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineCari">
            <property name="placeholderText">
             <string>Ketik untuk mencari...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCetak">
            <property name="text">
             <string>CETAK PDF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tblJemaat">
          <property name="columnCount">
           <number>9</number>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nama</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>JK</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Alamat</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tempat Lahir</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tgl Lahir</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>No HP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status Keluarga</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wilayah</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
